pages/reference/diagnostics.md

```diff
@@ -7,13 +7,16 @@ A `check` in this context is defined as a function that returns a result (good/b
 command is simply a data collection tool without any filtering or logic built in). Checks are intended to be used by
```

Comment:
Is this a good place to put the bracket or better to dispose of the bracket in this section entirely?
```suggestion
command is simply a data collection tool (without any filtering or logic built in). Checks are intended to be used by
```
based on @lurch's comment?

Reply:
Nice catch @lurch, I've opened a PR in the repo where we source this doc, and will pull that in once it's been approved.